nengo/tests/test_processes.py

```diff
@@ -99,7 +100,7 @@ def test_gaussian_whitenoise(Simulator, rms, seed, plt):
 def test_whitesignal_rms(Simulator, rms, seed, plt):
     t = 1.
     d = 500
-    process = WhiteSignal(t, rms=rms)
+    process = WhiteSignal(t, high=500, rms=rms)
```

Comment:
This isn't a huge deal, but since `high` is no longer a keyword argument I'd like to remove the `high=` everywhere we use WhiteSignal, in both tests and examples.